CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ librdkafka.
  * Added `rd_kafka_message_broker_id()` to see the broker that a message
    was produced or fetched from, or an error was associated with.
  * Added RTT/delay simulation to mock brokers.
+ * Added `sticky.partitioning.linger.ms` to improve batching of null-keyed messages
```

Comment:
This needs to go on the next release's (1.6.0) section, which is not yet available, so just add this line to the top of this file.